fcl/g4/larg4_icarus.fcl

```diff
@@ -104,4 +104,8 @@ physics.producers.mcreco.MCRecoPart.TrackIDOffsets: [0,10000000,20000000] #Accou
 physics.producers.sedlite.SimEnergyDepositLabel: "largeant:LArG4DetectorServicevolTPCActive"
 physics.producers.sedlite.useOrigTrackID: true #needed since origTrackID not filled for sedlite in SBND
 
+# avoid huge log files
+services.message.destinations.LogErrorFile.categories.SimDriftElectrons: { limit: 0 }
+services.message.destinations.LogStandardOut.categories.SimDriftElectrons: { limit: 0 }
```

Comment:
This should be fixed with LArSoft/larsim#152 (of course this means that it is _not yet_ fixed and that probably it won't be in the release we are targetting here...).